openupgrade_scripts/scripts/project/15.0.1.2/pre-migration.py

```diff
@@ -60,9 +60,22 @@ def adapt_project_task_dependency(env):
     )
 
 
+def rename_project_milestone_target_date(env):
+    """If project_milestone is installed then rename column target_date
+    to deadline.
+    """
+    if openupgrade.column_exists(env.cr, "project_milestone", "target_date"):
+        openupgrade.rename_columns(
+            env.cr,
+            {"project_milestone": [("target_date", "deadline")]},
+        )
+        # TODO: activate milestone in configuration
```

Comment:
This is done adding to internal user group the inherit.

Reply:
Ok, I will look at that. :)

Thank you very much.

Reply:
In fact, there is a configuration to activate milestone, but only in 16.0. Here we are migrating to 15.0, and milestone seams to be enabled by default. So there is nothing to do.